ecosystem/sep-0012.md

```diff
@@ -50,6 +50,20 @@ Alternatively, if the client cannot add the authorization header. The JWT should
 ?jwt=<token>
 ```
 
+## Content Type
+
+All endpoints accept in requests the following `Content-Type`s:
+
+- `multipart/form-data`
+- `application/x-www-form-urlencoded`
+- `application/json`
+
+All endpoints respond with content type:
+
+- `application/json`
+
+`multipart/form-data` should be used instead of `application/x-www-form-urlencoded` for requests transmitting binary data type values from [SEP-9](sep-0009.md).
```

Comment:
I think we should reword this a little to make it clear this is backwards compatible and you can continue to use the multipart content-type without binary data.
```suggestion
`multipart/form-data` should be used for requests including binary data type values from [SEP-9](sep-0009.md). `multipart/form-data` or `application/x-www-form-urlencoded` may be used when no binary fields are included.
```

I think this should be moved up to the request paragraph too because it's a little confusing for this to run after the comments about the response format.